src/Components/ImagesTable/ImagesTable.js

```diff
@@ -154,6 +158,37 @@ const ImagesTable = () => {
   const itemsStartInclusive = (page - 1) * perPage;
   const itemsEndExclusive = itemsStartInclusive + perPage;
 
+  const notificationProp = {
```

Comment:
https://github.com/RedHatInsights/image-builder-frontend/pull/1160 seems to add a copy of this, maybe just put this prop in `src/Utilities/edge.js` directly ? When 1160 gets rebased we won't have duplicate code then.

Reply:
@croissanne the src/Utilities/edge.js was added on #1160 can we do the clean up when rebase?